float128ppc/float128ppc.go

```diff
@@ -16,25 +16,25 @@ const (
 )
 
 var (
-	NegNaN = Float{a: -math.NaN(), b: 0}
-	NaN    = Float{a: math.NaN(), b: 0}
+	NegNaN = Float{high: -math.NaN(), low: 0}
+	NaN    = Float{high: math.NaN(), low: 0}
 )
 
-// Float is a floating-point number in double-double format.
+// Float is high floating-point number in double-double format.
```

Comment:
it seems like `a` was replaced with `high` in too many places ^^

Reply:
Oh damn, this feature of GoLand sometimes annoying when naming variable too casual XD.